components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/openidconnect/dao/RequestObjectDAOImplTest.java

```diff
@@ -20,30 +20,59 @@
 
 package org.wso2.carbon.identity.openidconnect.dao;
 
-import org.testng.Assert;
+import org.apache.commons.dbcp.BasicDataSource;
+import org.apache.commons.lang.StringUtils;
+import org.junit.Assert;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.testng.PowerMockTestCase;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 import org.wso2.carbon.identity.common.testng.WithCarbonHome;
 import org.wso2.carbon.identity.common.testng.WithH2Database;
-import org.wso2.carbon.identity.oauth2.IdentityOAuth2Exception;
+import org.wso2.carbon.identity.core.util.IdentityDatabaseUtil;
 import org.wso2.carbon.identity.openidconnect.model.RequestedClaim;
 
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.SQLException;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+
+import static org.mockito.Matchers.any;
+import static org.powermock.api.mockito.PowerMockito.doNothing;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.when;
 
 /**
  * This class contains unit tests for RequestObjectDAOImplTest..
  */
 @WithCarbonHome
 @WithH2Database(jndiName = "jdbc/WSO2IdentityDB",
         files = {"dbScripts/h2_with_application_and_token.sql", "dbScripts/identity.sql"})
-public class RequestObjectDAOImplTest {
+@PrepareForTest({IdentityDatabaseUtil.class})
+public class RequestObjectDAOImplTest extends PowerMockTestCase {
 
-    @Test
-    private void testInsertRequestObject() throws IdentityOAuth2Exception {
+    private final String sessionDataKey = "d43e8da324a33bdc941b9b95cad6a6a2";
+    private final String accessTokenId = "5b6ae5e6-08c2-36d4-9dfe-baff099ddb29";
+    private static final String DB_NAME = "testOpenid";
+
+    private RequestObjectDAO requestObjectDAO;
+    private List<List<RequestedClaim>> requestedEssentialClaims;
+
+    protected Connection connection;
+    protected BasicDataSource dataSource;
+    private static final Map<String, BasicDataSource> dataSourceMap = new HashMap<>();
+
+
+    @BeforeClass
+    public void setUp() throws Exception {
 
-        RequestObjectDAO requestObjectDAO = new RequestObjectDAOImpl();
-        List<List<RequestedClaim>> requestedEssentialClaims = new ArrayList<>();
-        List<RequestedClaim> lstRequestedCliams = new ArrayList<>();
+        requestObjectDAO = new RequestObjectDAOImpl();
+        requestedEssentialClaims = new ArrayList<>();
+        List<RequestedClaim> lstRequestedClams = new ArrayList<>();
```

Comment:
```suggestion
        List<RequestedClaim> lstRequestedClams = new ArrayList<>();
```
List<RequestedClaim> requestedClaims = new ArrayList<>();